src/index.js

```diff
@@ -0,0 +1,199 @@
+    state.sky[0].textContent = '💧☔️💧';
+  }
+
+  //CLOUDY or RAINY
+  state.cloud[0].classList.toggle('show', state.skySelect.value === 'cloudy' || state.skySelect.value === 'rainy');
+  state.cloud[1].classList.toggle('show', state.skySelect.value === 'cloudy' || state.skySelect.value === 'rainy');
+};
+
+const changeCityName = () => {
+  state.headerCityName.textContent = state.input.value;
+};
+
+const resetCity = () => {
+  state.input.value = '';
+  state.headerCityName.textContent = 'Primm';
+  updateWithRealtimeWeather()
+};
+
+const parseRegion = (region) => {
+  region = region.split(',');
+  region = `${region[0]}, ${region[region.length - 2]}, ${region[region.length - 1]}`;
+  return region;
+};
+
+const getLatLon = async () => {
+  const response = await axios.get('https://weather-report-proxy-msis.onrender.com/location', { params: { q: state.headerCityName.textContent } });
+  const { lat: latitude, lon: longitude } = response.data[0];
+  state.headerCityName.textContent = parseRegion(response.data[0].display_name);
+  return { latitude, longitude };
+};
+
+const setSkyWithRealtimeWeather = (code) => {
+  code = code.toString();
+  if (code.match(/2../) || code.match(/3../) || code.match(/5../)) {
+    state.skySelect.value = 'rainy';
+  }
+  else if (code.match(/6../)) {
+    state.skySelect.value = 'snowy';
+  }
+  else if (code.match(800)) {
+    state.skySelect.value = 'sunny';
+  }
+  else {
+    state.skySelect.value = 'cloudy';
+  }
+  changeSkyscape();
+};
+
+const getRealtimeWeather = async () => {
+  const { latitude, longitude } = await getLatLon();
+  const response = await axios.get('https://weather-report-proxy-msis.onrender.com/weather', { params: { lat: latitude, lon: longitude } });
+  const currTemp = response.data.main.temp;
+  const currSky = response.data.weather[0].id;
+  return { currTemp, currSky };
+};
+
+const updateWithRealtimeWeather = async () => {
+  const { currTemp, currSky } = await getRealtimeWeather();
+  state.temperature = Math.floor((currTemp - 273.15) * 9 / 5 + 32);
+  state.tempValue.textContent = state.temperature;
+  isCelcius();
+  setSkyWithRealtimeWeather(currSky);
+  changeColorBasedOnTemp();
+};
+
+const registerEvents = () => {
+  state.increaseTempControl.addEventListener('click', increaseTempBtn);
+  state.decreaseTempControl.addEventListener('click', decreaseTempBtn);
+  state.increaseTempControl.addEventListener('click', changeColorBasedOnTemp);
+  state.decreaseTempControl.addEventListener('click', changeColorBasedOnTemp);
+  state.currentTempButton.addEventListener('click', updateWithRealtimeWeather);
+  state.skySelect.addEventListener('change', changeSkyscape);
+  state.input.addEventListener('input', changeCityName);
+  state.resetButton.addEventListener('click', resetCity);
+  state.tempUnitToggle.addEventListener('change', isCelcius);
+  state.input.addEventListener('keyup', function onEvent(e) { if (e.keyCode === 13) { updateWithRealtimeWeather(); } });
```

Comment:
Nice event handler to perform the realtime weather update when the user hits enter in the city field.